src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
@HirdayGupta This approach is worse. Have you tried the wrapper div?

@amarlearning Note that @HirdayGupta changed it to absolute positioning - your screenshot shows inline block.